draft-rosa-bmwg-vnfbench

```diff
@@ -611,88 +725,74 @@ Internet-Draft                  VNFBench                   December 2018
-      routing table, firewall rules, subscribers profile).
+   Information about each component in a benchmarking setup (see
+   Section 5).  It contains the identification, name, image, role,
+   connection-points and resource requirements (i.e., allocation of cpu,
+   memory, disk).  Besides, in case of each node detaining the agent or
+   monitor role, its incumbent probers/listeners must be specified
+   detailing the experimental parameters to execute the benchmarking
+   tests, as the cases detailed below.
 
    Agents:   Defines the configured toolset of probers and related
       benchmarking/active metrics, available workloads, traffic formats/
       traces, and configurations to enable hardware capabilities (if
       existent).  In addition, it defines metrics from each prober to be
       extracted when running the benchmarking tests.
 
-
-
-
-Rosa, et al.              Expires July 2, 2019                 [Page 12]
-
-Internet-Draft                  VNFBench                   December 2018
-
-
    Monitors:   defines the configured toolset of listeners and related
       monitoring/passive metrics, configuration of the interfaces with
       the monitoring target (VNF and/or execution environment), and
       configurations to enable specific hardware capabilities (if
-      existent).  In addition, it defines metrics from each listener to
-      be extracted when running the benchmarking tests.
-
-6.1.4.2.  Environment
+      existent).  A monitor detains a specific set of fields detailing
+      its habitat, containing its setting (e.g., process, tap, etc),
+      location (internal or external) and target (e.g., execution
+      environment, VNF, etc).  So its listerners will be oriented to run
+      on this habitat.  In addition, it defines metrics from each
+      listener to be extracted when running the benchmarking tests.
+
+   The type is the most important field of a node.  If defined as an
+   internal one, it will be specified as depicted above.  However, if
+   specified as external, the field might refer to an outside file,
+   detailing a particular network service template to be parsed by the
+   orchestration plugin.
+
+   Lifecycle defines another node field.  It lists all the workflows
+   that must be realized for a node.  Main workflows include: create,
+   start, stop, delete.  Particular workflows can be specified
+   containing the required parameters and implementation.
+
+6.1.5.2.  Links
+
+   Information about the data plane links interconnecting the components
+   of a benchmarking setup.  Links refer to two or more connection-
+   points of a node.  A link might refer to be part of a network.
+   Depending on the link type, the network might be implemented as a
+   layer 2 mesh, or as directional-oriented traffic forwarding flow
+   entries.  Links also detain resource requirements, specifying the
+   minimum bandwidth, latency, and frame loss rate for the execution of
+   benchmarking Tests.
```

Comment:
Here People might ask why we specify the topology of the test in the BD itself, and do not simply re-use existing VNF/SFC description approaches as used by OSM/ONAP etc? Do we have an answer for this?

Reply:
Maybe make clear that topologies can either be defined BD or alternativly a topology descriptions can be referenced (external node concepts
)

Reply:
Rewrite the sentence to have a clear view of the external node being specified by an OSM/ONAP template.